test/executor.test.js

```diff
@@ -141,46 +150,55 @@ describe('executor', function() {
           sourceFile: undefined,
         },
       ],
-    }));
-
-    expect(Object.keys(app.models)).to.eql(['Car']);
+    }), function(err, context) {
+      if (err) return done(err);
+      expect(Object.keys(app.models)).to.eql(['Car']);
+      done();
+    });
   });
 
-  it('attaches models to data sources', function() {
-    boot.execute(app, dummyInstructions);
-    assert.equal(app.models.User.dataSource, app.dataSources.theDb);
+  it('attaches models to data sources', function(done) {
+    boot.execute(app, dummyInstructions, function(err, context) {
+      if (err) return done(err);
+      assert.equal(app.models.User.dataSource, app.dataSources.theDb);
+      done();
+    });
   });
 
-  it('defines all models first before running the config phase', function() {
-    appdir.writeFileSync('models/Customer.js', 'module.exports = ' +
-      function(Customer/*, Base*/) {
-        Customer.on('attached', function() {
-          Customer._modelsWhenAttached =
-            Object.keys(Customer.modelBuilder.models);
-        });
-      }.toString());
-
-    boot.execute(app, someInstructions({
-      models: [
-        {
-          name: 'Customer',
-          config: { dataSource: 'db' },
-          definition: { name: 'Customer' },
-          sourceFile: path.resolve(appdir.PATH, 'models', 'Customer.js'),
-        },
-        {
-          name: 'UniqueName',
-          config: { dataSource: 'db' },
-          definition: { name: 'UniqueName' },
-          sourceFile: undefined,
-        },
-      ],
-    }));
+  it('defines all models first before running the config phase',
+    function(done) {
```

Comment:
To reduce whitespace changes, we use the following indentation when fixing long lines:

```js
it('defines all models first before running the config phase',
function(done) {
  // inner code stays indented at the same level
});
```